hpx/runtime/threads/detail/scheduling_loop.hpp

```diff
@@ -822,10 +846,11 @@ namespace hpx { namespace threads { namespace detail
             // if nothing else has to be done either wait or terminate
             else
             {
-                ++idle_loop_count;
+                --idle_loop_count;
```

Comment:
Does this need to be reversed? Just thinking about naming. Right now it's more like `idle_loop_countdown` ;) and it's different from the other counts that actually grow.

Reply:
I reversed this as it helped me not to carry around the max value to the point where it needed to be reset.